src/librustdoc/lib.rs

```diff
@@ -404,6 +404,10 @@ fn opts() -> Vec<RustcOptGroup> {
 
 fn usage(argv0: &str) {
     let mut options = getopts::Options::new();
+    println!("rustdoc usage is documented online https://doc.rust-lang.org/rustdoc/");
+    if nightly_options::is_nightly_build() {
+        println!("nightly rustdoc includes additional features https://doc.rust-lang.org/nightly/nightly-rustc/rustdoc/");
+    }
```

Comment:
This should only link to one or the other I think.
```suggestion
    let url = if nightly_options::is_nightly_build() {
        "https://doc.rust-lang.org/nightly/nightly-rustc/rustdoc/"
    } else {
        "https://doc.rust-lang.org/rustdoc/"
    };
    println!("Documentation: {}", url);
```